lofo_importance.py

```diff
@@ -15,18 +15,17 @@ def __init__(self, model, df, features, target,
         self.scoring = scoring
         self.cv = cv
 
-    def _get_cv_score(self, X, y):
-        cv_results = cross_validate(self.model, X, y, cv=self.cv, scoring=self.scoring)
+    def _get_cv_score(self, feature_list):
+        cv_results = cross_validate(self.model, self.df[feature_list], self.df[self.target],
+                                    cv=self.cv, scoring=self.scoring)
         return cv_results['test_score']
 
     def get_importance(self):
-        lofo_cv_scores = []
+        base_cv_score = self._get_cv_score(self.features)
 
-        base_cv_score = self._get_cv_score(self.df[self.features], self.df[self.target])
-
-        for i, f in tqdm_notebook(enumerate(self.features)):
-            feature_list = [feature for feature in self.features if feature != f]
-            lofo_cv_scores.append(self._get_cv_score(self.df[feature_list], self.df[self.target]))
+        feature_lists = [[feature for feature in self.features if feature != f] for f in self.features]
+        pool = multiprocessing.Pool(len(self.features)+1)
```

Comment:
why  +1 ?

Reply:
Why not just None, to use all CPU
https://docs.python.org/2/library/multiprocessing.html#multiprocessing.pool.multiprocessing.Pool

Reply:
There's no real reason for the +1 except that I wanted to have more workers than jobs. I would prefer not to use None in case the user wants to run something else on the CPU.